src/services/AppUserService.ts

```diff
@@ -12,11 +16,31 @@ export class AppUserService {
   async saveAppUser(appUser: AppUser): Promise<AppUser> {
     const appUserRepository = getRepository(AppUser);
 
-    return await appUserRepository.save(appUser);
+    return appUserRepository.save(appUser);
+  }
+
+  /**
+   * Gets all app users.
+   *
+   * @returns {AppUser[]} Array of AppUser entities
+   */
+  getAllAppUsers(officerOnly: boolean, nameOnly: boolean): Promise<AppUser[]> {
+    const appUserRepository = getRepository(AppUser);
+    const query: FindManyOptions<AppUser> = {};
+
+    if (nameOnly) {
```

Comment:
later should have a helper private method that constructs the query based off query params

Reply:
sounds good